src/main/scala/org/zalando/nakadi/client/actor/PartitionReceiverActor.scala

```diff
@@ -34,9 +26,17 @@ object PartitionReceiver{
             automaticReconnect: Boolean,
             objectMapper: ObjectMapper) =
     Props(new PartitionReceiver(endpoint, port, securedConnection, topic, partitionId, parameters, tokenProvider, automaticReconnect, objectMapper) )
+
+  // Actor messages
+  //
+  private object Init
+  case class NewListener(listenerId: String, listener: ActorRef)
+  case class ConnectionOpened(topic: String, partition: String)
+  case class ConnectionFailed(topic: String, partition: String, status: Int, error: String)
+  case class ConnectionClosed(topic: String, partition: String, lastCursor: Option[Cursor])
```

Comment:
Moved these to companion object scope (instead of module scope). This involves using an `import` to use them, but I generally find it nice to enclose an actor's messages within the actor's scope. It kind of documents "these are my messages".